frontend/src/Student/queries/showForSubmission.graphql

```diff
@@ -0,0 +1,14 @@
+query Show($id: ID!) {
+  show(id: $id) {
+    id
+    name
+    entryCap
```

Comment:
This entry cap and comment look like their part of what Rob was doing in his PR? Do you mean to pull this info down here?

Reply:
what do you mean?

Reply:
I believe the updated shows.graphql query has what you're asking for. Why have you brought in this file? I don't see it used anywhere in this PR.

Reply:
Unsure. I will get rid of it